third_party/move/tools/move-cli/src/sandbox/utils/mod.rs

```diff
@@ -151,21 +151,10 @@ fn print_struct_diff_with_indent(
 
 pub(crate) fn explain_execution_effects(
     changeset: &ChangeSet,
-    events: &[Event],
     state: &OnDiskStateView,
 ) -> Result<()> {
     // execution effects should contain no modules
     assert!(changeset.modules().next().is_none());
-    if !events.is_empty() {
-        println!("Emitted {:?} events:", events.len());
-        // TODO: better event printing
-        for (event_key, event_sequence_number, _event_type, event_data) in events {
-            println!(
-                "Emitted {:?} as the {}th event to stream {:?}",
-                event_data, event_sequence_number, event_key
-            )
-        }
-    }
```

Comment:
Events intentionally removed from the "sandbox" (which I don't know what it is)? Is it okay?

Reply:
we are stripping out move events from the system, since there really isn't such a thing